src/api/namespaces/atomicmarket/handlers/sales2.get-sales-templates.test.ts

```diff
@@ -0,0 +1,423 @@
+            const {sale_id} = await client.createSale({
+                offer_id,
+                listing_price: 200000000,
+            });
+
+            expect(await getSalesIds({min_price: '2', burned: false}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('filters by maximum price', async () => {
+            const {template_id} = await initTest({
+                listing_price: 300000000,
+            });
+
+            const {offer_id} = await client.createOfferAsset({}, {template_id});
+            const {sale_id} = await client.createSale({
+                offer_id,
+                listing_price: 200000000,
+            });
+
+            expect(await getSalesIds({max_price: '2', burned: false}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('filters by collection', async () => {
+            const {template_id} = await initTest();
+
+            const {collection_name} = await client.createCollection({collection_name: 'x'});
+
+            const {offer_id} = await client.createOfferAsset({}, {template_id, collection_name});
+            const {sale_id} = await client.createSale({offer_id, collection_name});
+
+            expect(await getSalesIds({collection_name: 'x,abc'}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('filters by asset_id', async () => {
+            const {template_id} = await initTest();
+
+            const {asset_id, offer_id} = await client.createOfferAsset({}, {template_id});
+            const {sale_id} = await client.createSale({offer_id});
+
+            expect(await getSalesIds({asset_id}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('only returns single asset sales', async () => {
+            const {template_id, sale_id} = await initTest();
+
+            const {offer_id} = await client.createOfferAsset({}, {template_id});
+            await client.createOfferAsset({offer_id, index: 2}, {
+                template_id: (await client.createTemplate()).template_id,
+            });
+
+            await client.createSale({offer_id});
+
+            expect(await getSalesIds({burned: false}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('filters by asset owner', async () => {
+            const {template_id} = await initTest();
+
+            const {offer_id} = await client.createOfferAsset({}, {template_id, owner: 'x'});
+            const {sale_id} = await client.createSale({offer_id});
+
+            expect(await getSalesIds({owner: 'x,abc'}))
+                .to.deep.equal([sale_id]);
+        });
+
+        txit('filters by asset burned', async () => {
+            const {template_id} = await initTest();
+
+            const {offer_id} = await client.createOfferAsset({}, {template_id, owner: null});
+            const {sale_id} = await client.createSale({offer_id});
+
+            expect(await getSalesIds({burned: 'true'}))
```

Comment:
Now that I see this here, Why do you need to send the `burned` in all of the other cases?